packages/nextjs/components/Header.tsx

```diff
@@ -25,6 +25,10 @@ export const menuLinks: HeaderMenuLink[] = [
     href: "/debug",
     icon: <BugAntIcon className="h-4 w-4" />,
   },
+  {
+    label: "Lada",
+    href: "/builders/0xfd1F5B5F618313E19fA0eA3eEFAb422337AF4f99",
+  },
```

Comment:
There will be another task for linking the profiles on a better location, we can remove this link for now.

Reply:
You will be able to access your page by the browser address bar, typing the expected URL.

We are gonna add links to your page later, don't worry.